test/quantization/test_numeric_suite.py

```diff
@@ -160,3 +161,51 @@ def compare_and_validate_results(float_model, q_model, module_swap_list, data):
         self.assertTrue(isinstance(q_model.my_scalar_mul, Shadow))
         for k, v in ob_dict.items():
             self.assertTrue(v["float"].shape == v["quantized"].shape)
+
+    @unittest.skipUnless(
```

Comment:
Please ensure that the tests pass for QNNPACK also. See changes to existing tests (in test_numeric_suite) for information on how to do this.

Reply:
Yes, I have added the same decorator as the existing tests have.